README.md

```diff
@@ -1,8 +1,4 @@
 # active-task-manager
 active-task-manager
 
-
-
-
-
-
+<p><blockquote>Express is a light-weight web application framework to help organize your web application into an MVC architecture on the server side. TypeScript enables us to develop a solid web applications using Express on Node.js.</blockquote></p>
```

Comment:
@oscarmild 

Why dont you complete the whole readme.?

Reply:
@fromgopi We will write by next pr